Energy Efficiency EACs Using Pre and Post Measurements.md

```diff
@@ -0,0 +1,516 @@
+
+
+
+* Selection of representative circuits to isolate lighting loads being retrofit
+* Installation of meters to record amperage or power in no more than 15-minute intervals
+* Storage of the interval data for analysis on a centralized server 
+* Performing calculations of energy savings and reporting results in a web-based or API based platform
+* A fixture sampling plan that achieves a 10% precision at 90% confidence covering fixtures retrofit, assuming a Coefficient of Variation of 0.5 as calculated below
+
+First determine the initial sample size; 
+```math 
+n = \frac {Z^2CV^2} {P^2}
+```
+    where; 	
+    n = initial sample size 
+    Z = The z-statistic for the desired level of confidence. Equal to 1.645 for a confidence level of 90% 
+    CV =assumed coefficient of variation (0.5) 
+    P = desired precision (10%)
+
+
+and then determine the adjusted sample size based on the population; 
+```math 
+n* = \frac {Nn} {N+n}
+```
+    where; 	
+    n* = adjusted sample size (actual minimum sample fixtures) 
+    n = initial sample size (calculated above) 
+    N = population size (total fixtures)
+
+Resulting in the following minimum sample sizes based on total fixtures to achieve a 10% precision at 90% confidence:
+
+
+<table>
+  <tr>
+   <td><strong>Total Fixture Quantity Retrofit (N)</strong>
+   </td>
+   <td><strong>Minimum Sample Fixture Quantity (n*)</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>3
+   </td>
+   <td>                         3 
+   </td>
+  </tr>
+  <tr>
+   <td>5
+   </td>
+   <td>                         5 
+   </td>
+  </tr>
+  <tr>
+   <td>10
+   </td>
+   <td>                         9 
+   </td>
+  </tr>
+  <tr>
+   <td>25
+   </td>
+   <td>                       19 
+   </td>
+  </tr>
+  <tr>
+   <td>50
+   </td>
+   <td>                       29 
+   </td>
+  </tr>
+  <tr>
+   <td>100
+   </td>
+   <td>                       41 
+   </td>
+  </tr>
+  <tr>
+   <td>250
+   </td>
+   <td>                       54 
+   </td>
+  </tr>
+  <tr>
+   <td>500
+   </td>
+   <td>                       60 
+   </td>
+  </tr>
+  <tr>
+   <td>1000
+   </td>
+   <td>                       64 
+   </td>
+  </tr>
+  <tr>
+   <td>2500
+   </td>
+   <td>                       66 
+   </td>
+  </tr>
+  <tr>
+   <td>5000
+   </td>
+   <td>                       67 
+   </td>
+  </tr>
+  <tr>
+   <td>≥7500
+   </td>
+   <td>                       68 
+   </td>
+  </tr>
+</table>
+
+
+Where baseline data logging is conducted, the same circuits will be monitored post retrofit for side-by-side comparison.  In such cases, the difference between measured post energy consumption to baseline energy consumption will be used to verify the true reduction;
```

Comment:
What do you mean by "verify"? How do the measured values inform the overall savings?